include/rcheevos.h

```diff
@@ -171,9 +192,6 @@ enum {
 typedef struct rc_condition_t rc_condition_t;
 
 struct rc_condition_t {
-  /* The next condition in the chain. */
-  rc_condition_t* next;
```

Comment:
putting this before the `rc_operand_t` structs allocated an extra buffer between `next` and `operand1` so `operand1` would be 8-byte aligned.